omero/sysadmins/version-requirements.txt

```diff
@@ -0,0 +1,1883 @@
+  number of 32-bit Windows clients, so support should probably be
+  retained. However, new systems are almost all 64-bit so dropping
+  32-bit support is a possibility for 5.2.
+
+Components
+,,,,,,,,,,
+
+* PostgreSQL
+
+  * Proposal: [5.1] 9.2 recommended, 9.3+ supported, 8.4 dropped, 9.0
+    and 9.1 also dropped
+  * Proposal: [5.2] 9.3 recommended, 9.4+ supported, 9.2 supported or
+    dropped
+  * Rationale: Current releases available for all supported systems
+    from upstream and for CentOS/RHEL 6.x officially via SCL (software
+    collections). At a bare minimum, 8.4 and 9.0 must be dropped (end
+    of security support); dropping 9.1 removes the floating point
+    issues fixed by 9.2+. Due to its age, 8.4 no longer receives
+    adequate testing and so we can’t claim to support it. The same
+    applies to 9.0 and 9.1.
+
+* Python
+
+  * Proposal: [5.1] 2.7 recommended, 2.6 supported, (3.x not
+    supported)
+  * Proposal: [5.2] 2.7 recommended, 2.6 dropped, (3.x not supported)
+  * Rationale: 2.7 is provided by all systems except for CentOS/RHEL
+    6.x, however it is available officially via SCL so could be made
+    the minimum. Note that 3.3 is also available via SCL, so there is
+    potential for a 3.x migration in the 5.2 timeframe. Dropping 2.6
+    would unify python support to a single major version, and remove a
+    number of portability issues; now it is restricted to CentOS/RHEL
+    6, in practice it is much less tested than 2.7 since no developer
+    is using it routinely. Removing 2.6 support will be dependent upon
+    internal testing of the SCL python in CentOS.
+
+* GCC
+
+  * Proposal: [5.1] 4.4
+  * Proposal: [5.2] Depends upon CentOS6 and Ubuntu 12.04 support
+    status
+  * Rationale: 4.4 is the CentOS/RHEL 6.x default. It would be
+    desirable to update but is probably not what external users will
+    want given the compatibility concerns. The SCL DeveloperToolset3
+    could bring this up to GCC 4.9.1, which would make the new
+    baseline GCC 4.6 (Ubuntu 12.04 and Travis)
+
+* LLVM/clang
+
+  * Proposal: 3.4 
+  * Rationale: It is the current toolchain on MacOS/FreeBSD and is
+    working well; 3.3 dropped due to no longer being used by current
+    toolchains
+
+* MSVC
+
+  * Proposal: [5.1] 2010
+  * Proposal: [5.2] 2012
+  * Rationale: The minimum (and maximum) MSVC version is determined by
+    the Windows Ice builds provided by ZeroC, and as such is limited
+    by our Ice version requirements. Raising the limit in 5.2 will
+    permit use of newer language features.
+
+* Ice
+
+  * Proposal: [5.1] 3.4 supported, 3.5 recommended, 3.6 likely not
+    supported
+  * Proposal: [5.2] 3.4 dropped, 3.5 supported, 3.6 recommended
+    (subject to testing)
+  * Rationale: 3.4 is no longer supported by ZeroC with 3.5 being
+    current. By the release of 5.2, Ice 3.6 will have been out for
+    some time and should be well tested.
+
+* Java
+
+  * Proposal: [5.1] 7 recommended, 8 supported, 6 dropped
```

Comment:
Does that imply we need to be damn sure the 5.2 **Recommended** versions work properly in 5.1?